lsusb.c

```diff
@@ -320,6 +322,59 @@ static void dump_device(
 	free(serial);
 }
 
+
+/* Dump only serial number as text (mode==0) or hexadecimal (mode!=0) */
+static void dump_serial(libusb_device *dev, struct libusb_device_descriptor *desc, int mode)
+{
+	libusb_device_handle *udev;
+	if(!desc->iSerialNumber) {
```

Comment:
Please follow the kernel coding style, add a ' ' after "if" and before "(" please.

Same thing elsewhere in this function.

Reply:
OK. Sorry. Normally I don't use this space...

Reply:
I've added the missing spaces, changed/removed comment and fixed a typo.